charts/quickwit/templates/metastore-deployment.yaml

```diff
@@ -46,11 +46,20 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}        
           args: ["run", "--service", "metastore"]
           env:
-            {{- include "quickwit.metastore.environment" . | nindent 12 }}
+            {{- include "quickwit.environment" . | nindent 12 }}
             {{- range $key, $value := .Values.metastore.extraEnv }}
             - name: "{{ $key }}"
               value: "{{ $value }}"
             {{- end }}
+          {{- if or (.Values.environmentFrom) (.Values.metastore.extraEnvFrom) }}
+          envFrom:
```

Comment:
See this comment: https://github.com/quickwit-oss/helm-charts/pull/85#issuecomment-2176463986